ui/tests/utils/generate-raw-json-job.js

```diff
@@ -0,0 +1,549 @@
+                IP: '12.228.2.247',
+                MBits: 10,
+                Mode: 'bridge',
+                ReservedPorts: [],
+                DynamicPorts: [],
+              },
+              {
+                Device: 'eth1',
+                CIDR: '',
+                IP: '71.92.163.164',
+                MBits: 10,
+                Mode: 'bridge',
+                ReservedPorts: [],
+                DynamicPorts: [],
+              },
+            ],
+            Ports: [
+              {
+                Label: 'pixel',
+                Value: 32161,
+                To: 25929,
+                HostIP: 'b778:a055:4422:e3ca:fd2d:e5e8:464d:8f2b',
+              },
+            ],
+          },
+          Resources: {
+            CPU: 2000,
+            MemoryMB: 1024,
+            MemoryMaxMB: 0,
+            DiskMB: 0,
+          },
+          Driver: 'qemu',
+          Name: 'task-microchip-3',
+          JobID: '',
+          VolumeMounts: [
+            {
+              Volume: 'mazie',
+              Destination: '/Vella.OReilly/dudley/#4b403a',
+              PropagationMode: '',
+              ReadOnly: true,
+            },
+            {
+              Volume: 'leora',
+              Destination: '/Tressa_Brown/ian/#6d666c',
+              PropagationMode: '',
+              ReadOnly: false,
+            },
+          ],
+          GroupNames: [],
+          WithServices: false,
+          CreateRecommendations: true,
+          ID: '4',
+          Services: [],
+        },
+      ],
+    },
+  ],
+  JobSummary: {
+    GroupNames: ['pixel-g-0', 'protocol-g-1'],
+    Summary: {
+      'pixel-g-0': {
+        Queued: 10,
+        Complete: 10,
+        Failed: 6,
+        Running: 7,
+        Starting: 4,
+        Lost: 8,
+        Unknown: 1,
+      },
+      'protocol-g-1': {
+        Queued: 4,
+        Complete: 7,
+        Failed: 2,
+        Running: 0,
+        Starting: 7,
+        Lost: 3,
+        Unknown: 5,
+      },
+    },
+    Namespace: 'namespace-1',
+    ID: '1',
+    JobID: 'hdd-panel-0',
+  },
+  Specification: {
+    Definition:
+      'job "docs" {\n  namespace = "madness"\n  group "example" {\n    task "server" {\n      service {\n        tags = ["leader", "mysql"]\n\n        port = "db"\n\n        meta {\n          meta = "for your service"\n        }\n\n        check {\n          type     = "tcp"\n          port     = "db"\n          interval = "10s"\n          timeout  = "2s"\n        }\n\n        check {\n          type     = "script"\n          name     = "check_table"\n          command  = "/usr/local/bin/check_mysql_table_status"\n          args     = ["--verbose"]\n          interval = "60s"\n          timeout  = "5s"\n\n          check_restart {\n            limit = 3\n            grace = "90s"\n            ignore_warnings = false\n          }\n        }\n      }\n    }\n  }\n}\n',
+    Type: 'hcl',
+    Variables: {
+      datacenters: ['west'],
+      external_port: 4000,
+    },
+  },
+};
```

Comment:
I'd rather this be generated in Mirage rather than have its JSON pasted verbatim. If we want to vary this test and expand on it later, it would make it easier to do if we could modify it at mirage server.schema level.

Reply:
Deleted my previous comment. The reason that this file is here is because of `fetchRawRequest`. This is what hydrates `CodeMirror` previously we never tested how this API request was responsible for hydrating this view to ensure that its correct. Take a look into the acceptance test for `Job Definition` to see how this is being used.

The entire purpose of `fetchRawRequest` is that its dispatching a request using an adapter, however its side stepping the Ember Data store. Its the same request URL as fetching a job, however, we don't want to serialize the API response and thus we must override the default behavior of Mirage. To accomplish this, we use a passthrough recipe where we call a `pretender` method instead of a Mirage request to ensure that we get a proper `JSON` response.

This is Mirage/Pretender pattern and this is how we handle the evaluation response in the `Evaluations` view.

I believe this is the correct pattern for the acceptance test.